.github/boring-cyborg.yml

```diff
@@ -131,6 +130,8 @@ labelPRBasedOnFilePath:
         - extensions/infinispan-client/**/*
         - integration-tests/infinispan-cache-jpa/**/*
         - integration-tests/infinispan-client/**/*
+    area/infra:
+        - .mvn/**/*
```

Comment:
Yeah so I refrain from doing that because that would mean any PR including changes to these files would be marked with `infra` and thus out of orphan PRs detection.

That's the main usage of this label.

Reply:
Hm, why you added that label in https://github.com/quarkusio/quarkus/pull/11615 then?

Reply:
Because it was only touching this file.

The issue is that you can't say "add this label if you only touch this directory".

As soon as you add the `infra` label, it's considered a PR that is not part of a release. It will be true for PRs only touching this directory but incorrect for PRs touching this one and an extension for instance.